docs/source/app_user_manual.rst

```diff
@@ -20,52 +20,115 @@ the app should be launched from the code directory.
+#. Continuous
+	- Continuous variables are values that are doubles, floats, or integers. These are usually numeric values.
+
+Setting Variable Roles
+***********************
+There are five different possible roles for a variable.
+
+#. Splitby
+	- Splitby variables are generally categories that you want to split by and analyze, such as categorical or ordinal variables.
+	- Common examples are gender, race, or age group. 
+#. Independent
+	- Independent variables are those that do not depend on another variable, or one that you wish to visualize on the y-axis.
+#. Dependent
+	- Dependent variables are the variables that you wish to measure based on the independent variables.
+	- Common examples are time or other continuous variables.
+#. Prediction
+#. Groundtruth
+#. Ignore
+	- Ignore variables are those that the user does not want to consider for calculations or visualization. These can be variables that are irrelevant in consideration of the mix effect, or variables that the user wants to use as a weighting variable.
+
+Other Metadata Settings
+************************
+#. isCount
+	- isCount can be used for ignore variables. It is most useful whenever you have a population variable, or a 'count' variable, that you want
+	to use as a weight for certain categories or trend variables. In order to use a variable as isCount, mark the variable role as 'ignore' then select 'Y' for isCount.
+#. weighting_var
+	- In order to utilize an ignore, isCount variable properly, it must be set as the weighting_var, or weighting variable, for at least one other variable in the metadata table.
+	The weighting_var can be set to any variable in the table, but is best utilized if it is an ignore, isCount variable.
+#. Example
 
 Data Augmentation
 ^^^^^^^^^^^^^^^^^^
 
+Data augmentation is useful for you if you want to combine certain categorical variables to analyze. In the metadata entry screen, there is a checkbox for quantiles and intersection for each variable.
+Intersection is used for categorical variables. 
+
+Checking the intersection checkbox for more than one categorical variable creates a new column in your data representing the intersection of the two variables.
+For example, if you wanted to discover trends related to a categorical combination like 'White Female', check the intersection box for the variables race and gender.
+
+
+Check the quantile box for continuous variables that you want to discretize.
+
 Choosing Trends
 ^^^^^^^^^^^^^^^^^
+There are multiple trends that you can choose from to analyze your data. 
+
+Pearson Correlation
+********************
+Pearson Correlation is measure of linear correlation between two variables. It includes a numerical co-efficient ranging form -1.0 to +1.0. The co-efficient is used to determine how positively or negatively the association between the two variables are.
+Rank Trend
+***********
+
```

Comment:
Rank trend is used to measure ordinal association. It has a coefficient that measures he degree of similarity between two rankings, and it can be used to assess the significance of the relationship between them.